src/orquestra/quantum/circuits/_wavefunction_operations.py

```diff
@@ -80,9 +81,9 @@ def free_symbols(self) -> Iterable[sympy.Symbol]:
         - an `H` gate has no free symbols
         - a `RX(np.pi)` gate has no free symbols
         - a `RX(sympy.Symbol("theta"))` gate has a single free symbol `theta`
-        - a `RX(sympy.sympify("theta * alpha"))` gate has two free symbols, `alpha` and
+        - a `RX(sympy.simplify("theta * alpha"))` gate has two free symbols, `alpha` and
```

Comment:
I think it actually should have been `sympify`. Have you checked whether it works?

Reply:
I checked whether symplify was a sympy function and it didn't come up. So I figured it was just a typo.
But now that I see the function sympify is a sympy function.

Reply:
👌